### PR TITLE
Dr 23 11 sinai new et002

### PR DESCRIPTION
**Pull request by thea-m**
Some questions are in the comments. The text of the addition is partly still not understandable to me (maybe it is not all Geez). I didn't suggest yet a date for the hand (my feeling is rather the beginning of the period in question, but I leave it to you)

### CODE REVIEW

File: SinaiStCatherine/SinaiNewEt002.xml [hunk 1]
```diff
@@ -184,6 +368,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                          <locus target="#1r"/>
                          <desc type="Unclear">Note in the upper margin.</desc>
                          <q xml:lang="gez">ዐጋቶናማ፡ ኢሳይያስ፡ ትሬያዶስ፡ <gap reason="illegible"/>ዎቶቁ፡ በይቲ፡ ይሚ<gap reason="illegible"/>ንስ።</q>
+                         <!--create person IDs?-->
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply SinaiStCatherine/SinaiNewEt002.xml:371]: But what is the language? The note is not understandable. Please reflect this in the note.

File: SinaiStCatherine/SinaiNewEt002.xml [hunk 2]
```diff
@@ -207,33 +392,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                         sections are written to the left of the column.</desc>
+                       </item>
                       </list>
                   </additions>
 
                   <bindingDesc>
                      <binding contemporary="non-Ethiopian" xml:id="binding">
-                        <decoNote xml:id="b1">Islamic binding, boards of paper or cardboard covered with blind-tooled leather, with flap.</decoNote>
-                        <decoNote xml:id="b6" type="bindingMaterial">
+                        <decoNote xml:id="b1">Islamic binding, boards of paper or cardboard covered with blind-tooled leather, with flap. A small leather strip is inserted into the hole in the flap.</decoNote>
```
Comment (DenisNosnitsin1970) [inline SinaiStCatherine/SinaiNewEt002.xml:424]: Perhaps, we should cautiously say: "Non-Ethiopian binding"

File: SinaiStCatherine/SinaiNewEt002.xml [hunk 3]
```diff
@@ -207,33 +392,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                       <decoNote xml:id="b2" type="Cover">Blind-tooled brown leather cover.</decoNote>
```
Comment (DenisNosnitsin1970) [inline SinaiStCatherine/SinaiNewEt002.xml:425]: If you look closer, you will see that the multiple rosettes have the same decorativ motif inside, smth like bird (cock?)

File: SinaiStCatherine/SinaiNewEt002.xml [hunk 4]
```diff
@@ -207,33 +392,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                         <q xml:lang="gez">ሀይጣሕል፡ ዝንቱ፡ መጸሐፈ፡ ንሳዕዐ</q>
+                         <locus target="#64v"/>
+                         <desc type="Unclear">Written in the upper margin in red ink.</desc>
+                         <q xml:lang="gez">ቶማስ፡</q> <!--create person ID?-->
                        </item>
                        <item xml:id="a7">
                          <locus target="#110v"/>
-                         <desc type="Unclear">Note in the lower margin, possibly continuing <ref target="#a6"/>.</desc>
-                         <q xml:lang="gez">ዐበጊ፡ ዘወተር፡ ሀይጣሕል፡ ከተከከ<gap reason="lost"/>ክርስቶስ፡ ሀተታዐምር</q>
+                         <desc type="Unclear">Note in the upper margin, partly incomprehensible.</desc>
```
Comment (DenisNosnitsin1970) [inline SinaiStCatherine/SinaiNewEt002.xml:401]: I would like to have a look at the notes, I would need the numbers of images, to come through recto - verso sets :)

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: SinaiStCatherine/SinaiNewEt002.xml [hunk 5]
```diff
@@ -157,8 +315,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                          </dimensions>
                          <ab type="ruling">Ruling is visible.</ab>
                          <ab type="pricking">Pricking is not visible.</ab>
-                         <ab type="ruling" subtype="pattern"></ab>
-                         
+                         <!--<ab type="ruling" subtype="pattern"></ab>-->
+                         <ab type="ruling">The upper line is written above the ruling.</ab>
```
[the remark this answers is not in the document]
Comment (thea-m) [reply SinaiStCatherine/SinaiNewEt002.xml:319]: I gave examples. In general, I find the horizontal ruling more visible on the verso than on the recto

File: SinaiStCatherine/SinaiNewEt002.xml [hunk 6]
```diff
@@ -167,14 +329,36 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                       <!-- <seg type="script">Second half of the 14th-/early 15th-century script.</seg>-->
-                      <seg type="ink">Black, red.</seg>
-                      <seg type="rubrication"></seg>
-                      <desc></desc>
+                      <seg type="ink">Black.</seg>
+                      <desc>The loops of <foreign xml:lang="gez">መ</foreign> are not separated (<locus target="#1r #2r"/>);  
+                        the vowel marker of <foreign xml:lang="gez">ስ</foreign> is composed of a vertical line and a horizontal line directed to the left(<locus target="#1r"/>);
+                        the loop of <foreign xml:lang="gez">ሎ</foreign> is directly attached to the upper part of the body (<locus target="#1r #111r"/>);
+                        the two arms of <foreign xml:lang="gez">ጥ</foreign> extend nearly to the bottom of the character <locus target="#1v"/>;
+                        the vowel markers of <foreign xml:lang="gez">ዝ</foreign> and <foreign xml:lang="gez">ት</foreign> are attached to the body of the character diagonally (<locus target="#2r"/>). </desc>
```
[the remark this answers is not in the document]
Comment (thea-m) [reply SinaiStCatherine/SinaiNewEt002.xml:361]: ስ is written differently on f. 1r (rather two lines) and 1v and 110v-111r (one curved line). Maybe the hand is not the same? My impression was that it was one hand (see lo and tu, for example), but that might be wrong. The ink also seems to be different.

**Comment by eu-genia** [comment]
nb the file is not yet valid (I am sure you know)

**Comment by thea-m** [comment]
Thank you, yes, it waited for the merging of person PR, done now

**Pull request merged**